src/domains/login/common/components/CustomDomain.jsx

```
@@ -0,0 +1,16 @@
+const CustomDomain = ({ customDomain, setCustomDomain,disabled }) => {
```

ESLint

[error] 'customDomain' is missing in props validation

(react/prop-types)